colorization/colorize_process.py

```diff
@@ -10,22 +10,18 @@
 
 def preprocess(image_file):
     """
-    This function reads the imageFile as a float-Matrix;
-    downsize to modelWidth*modelHeight;
-    if the process run in Atlas, copys the picture data to the device;
-    copys the L channel into the malloc memory location.
+    This function downsizes the input image to modelWidth*modelHeight
+    and extracts L_channel from the image.
 
     Parameters:
     -----------
     input:
-    imageFile : str
+    image_file : str
```

Comment:
this is also a numpy array I think

Reply:
I agree, it should be a numpy array

Reply:
yes, correct it should be a NumPy array. I have updated it now.